src/test/java/duke/tasks/EventTest.java

```diff
@@ -0,0 +1,58 @@
+package duke.tasks;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.temporal.ChronoField;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class EventTest {
+    private static final DateTimeFormatter FORMATTER = new DateTimeFormatterBuilder()
+            .appendPattern("yyyy-MM-dd")
+            .optionalStart().appendPattern(" HH:mm").optionalEnd()
+            .optionalStart().appendPattern(" HHmm").optionalEnd()
+            .parseDefaulting(ChronoField.HOUR_OF_DAY, 0)
+            .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
+            .toFormatter();
+    private Event event;
+
+    @BeforeEach
+    public void init() {
+        this.event = new Event("do cs2103t",
+                LocalDateTime.parse("2023-08-30 22:19", FORMATTER),
+                LocalDateTime.parse("2023-09-20", FORMATTER));
+    }
+
+    @Test
+    public void testStringConversion() {
```

Comment:
A note that according to the Java coding standard, names like "stringConversion" would suffice as test methods for the method it is meant to test e.g. "stringConversion"